pkg/component/networking/vpn/shoot/shoot_test.go

```diff
@@ -739,6 +740,13 @@ var _ = Describe("VPNShoot", func() {
 					}...)
 				}
 
+				if features.DefaultFeatureGate.Enabled(features.VPNBondingModeRoundRobin) {
```

Comment:
We could accept it as arg of the func instead of duplicating the same code in the test.